regras.html

```
@@ -0,0 +1,658 @@
+        <span>🎯 Objetivo: último com pontos</span>
+      </div>
+    </header>
+
+    <!-- Setup -->
+    <div class="section-title">Setup & Objetivo</div>
+    <div class="card">
+      <div class="card-header">
+        <div class="card-number">01</div>
+        <h2>Objetivo do Jogo</h2>
+      </div>
+      <p>Ser o <strong>último jogador com pontos positivos</strong>. Todos começam com <strong>5 pontos</strong>. Os pontos apenas <strong>diminuem</strong>. O jogo acaba quando só um jogador tem pontos > 0.</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)